content/plugins/overview.md

```diff
@@ -17,60 +17,9 @@ To cover the two major types of plugins (IaaS and Management tools), let's also
 
 # Plugin Development
 
-You should check out the [Plugins Authoring Guide](plugins-authoring.html) if you want to write your own plugin for your chosen tool or IaaS provider.
+You should check out the [Plugins Authoring Guide]({{< relref "plugins/creating-your-own-plugin.md" >}}) if you want to write your own plugin for your chosen tool or IaaS provider.
 
-The Python module which provides the API for a plugin to interact with Cloudify is called the cloudify-plugins-common module.
-The module provides features for getting and setting context, downloading blueprint resources and much more and its reference can be found [here](apis-plugins-common.html).
+In order to learn how to create and use a plugin package, see the [plugins usage guide]({{< relref "plugins/using-plugins.md" >}})
```

Comment:
Again, please let's not name this "The X Guide"..

maybe.. see the "using plugins" page? :)